### PR TITLE
Support setting rdp timeouts via envars

### PR DESCRIPTION
**Pull request by richlanc**
Included:
* Increase the default FreeRDP timeout variables
* Make these timeouts configurable via environment variables.

Testing:

The machine we failed to connect to had a RTT of 600ms, so let's artificially introduce latency.

Introduce latency via (latency is twice as bad as the customer env): 
```sh
$ toxiproxy-server &
$ toxiproxy-cli create -l 127.0.0.1:3389 -u <rdp-host-up>:3389 rdp
$ toxiproxy-cli toxic add  -u -t latency -a latency=600 rdp
$ toxiproxy-cli toxic add  -d -t latency -a latency=600 rdp
```

This setup should fail to work without the change in this PR.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -193,13 +198,6 @@ USER guacd
 # Expose the default listener port
 EXPOSE 4822
 
-# Start guacd, listening on port 0.0.0.0:4822
-#
-# Note the path here MUST correspond to the value specified in the
-# PREFIX_DIR build argument.
-#
-CMD /opt/guacamole/sbin/guacd -b 0.0.0.0 -L $GUACD_LOG_LEVEL -f
```
Comment (SpeeddY) [inline Dockerfile:201]: Why has this been removed  ?

**Review by SpeeddY — CHANGES_REQUESTED** [review]
Do we have a README where we keep track of our patches ? 
Could we have a README update if we do

File: Dockerfile (hunk 1 above)
Comment (richlanc) [reply Dockerfile:201]: It confused me, when I was messing with log versions. 

We actually call guacd from guacd.sh

File: src/protocols/rdp/rdp.c [hunk 2]
```diff
@@ -515,6 +526,9 @@ static int guac_rdp_handle_connection(guac_client* client) {
     /* Set default pointer */
     guac_common_cursor_set_pointer(rdp_client->display->cursor);
 
+    rdp_inst->settings->TcpAckTimeout = get_envvar_uint32("GUACD_RDP_TCP_ACK_TIMEOUT", 30000);  // FreeRDP default 9000
```
Comment (SpeeddY) [inline src/protocols/rdp/rdp.c:529]: if the default is `9000` (so 9s?) 
How did it manage to fail with a round-trip time of 600 ms ?

File: Dockerfile (hunk 1 above)
Comment (SpeeddY) [reply Dockerfile:201]: right I see :+1:

File: src/protocols/rdp/rdp.c (hunk 2 above)
Comment (richlanc) [reply src/protocols/rdp/rdp.c:529]: I'm not sure, I didn't dig any deeper after changing it on a hunch and it resolved the issue reliably.

**Review by SpeeddY — APPROVED** [review]
works great, tested with toxiproxy as detailed in the description

**Pull request merged**